rakelib/run_rspec.rake

```diff
@@ -17,11 +17,6 @@ namespace :run_rspec do
     run_tests_in("spec/dummy", env_vars: "DRIVER=selenium_firefox")
```

Comment:
heads up we're overriding the env var here, we should just delete this and rely on the rails_helper?